dashboard/src/t5gweb/libtelco5g.py

```diff
@@ -425,7 +425,8 @@ def generate_stats(account=None):
     if account is not None:
         logging.warning("filtering cases for {}".format(account))
         cards = {c: d for (c, d) in cards.items() if d["account"] == account}
-        cases = {c: d for (c, d) in cases.items() if d["account"] == account}
+        if engineer is None:
+            cases = {c: d for (c, d) in cases.items() if d["account"] == account}
```

Comment:
This logic results in an incorrect "Open Case" count in the stats table. Instead of showing the number of cases that the engineer is working on, it displays the total number of cases. 

I've been trying this, but the numbers aren't adding up:
```python
    if account is not None:
        logging.warning("filtering cases for {}".format(account))
        cards = {c: d for (c, d) in cards.items() if d["account"] == account}
        cases = {c: d for (c, d) in cases.items() if d["account"] == account}
    if engineer is not None:
        # get case number and assignee from cards so that we can determine which cases
        # belong to the engineer
        logging.warning("filtering cases for {}".format(engineer))
        cards = {
            c: d for (c, d) in cards.items() if d["assignee"]["displayName"] == engineer
        }
        temp_cases = {}
        for case, details in cases.items():
            for card in cards:
                if (
                    case == cards[card]["case_number"]
                    and cards[card]["assignee"]["displayName"] == engineer
                ):
                    temp_cases[case] = details
        cases = temp_cases
```

in any case, this might be a starting point for a solution.

Reply:
> 
> I've been trying this, but the numbers aren't adding up (the number of "Open Cases" in the table does not match the open cards in the table below):
> 
> 
> in any case, this might be a starting point for a solution.

The tables includes both open and closed cards/cases though. Could that be the reason for the mismatch you're seeing?

Reply:
My numbers aren't in sync with production, but the logic should be the same: I'm seeing 85 cases in the table, 20 of which are closed. So under Open cases we should see 85-20 =65, but I'm seeing 44 instead

Reply:
@dcritch OK, I think I narrowed down the issue: 
1. I think the numbers from my earlier comment are off, so after refreshing:
2. The stats tables are populated using `generate_stats()`, while the interactive tables at the bottom are generated by `get_new_comments()`. `get_new_comments()` throws out cards that don't have any comments on them, while `generate_stats()` doesn't, which results in different numbers between the top and bottom halves of the page.

In production, it's quite rare for a card to have no comments, so the mismatch doesn't really show up anywhere. I think we implemented this for the reports (`<URL>updates/all`) so that every bullet would have some content, and then just plugged it into the tables when we were creating the table view. Should we remove the behavior of dropping cards with 0 comments to make everything synced up with each other?

TL;DR I believe the mismatch is outside of the scope of this PR. @ptrnull When you get a chance, could you update your branch with the suggestion mentioned above (repeated below as well for convenience):

libtelco5g.py:425 =>
```python
    if account is not None:
        logging.warning("filtering cases for {}".format(account))
        cards = {c: d for (c, d) in cards.items() if d["account"] == account}
        cases = {c: d for (c, d) in cases.items() if d["account"] == account}
    if engineer is not None:
        logging.warning("filtering cases for {}".format(engineer))
        cards = {
            c: d for (c, d) in cards.items() if d["assignee"]["displayName"] == engineer
        }

        # get case number and assignee from cards so that we can determine which cases
        # belong to the engineer
        temp_cases = {}
        for case, details in cases.items():
            for card in cards:
                if (
                    case == cards[card]["case_number"]
                    and cards[card]["assignee"]["displayName"] == engineer
                ):
                    temp_cases[case] = details
        cases = temp_cases
```

Reply:
great detective work! yes, maybe we should return 'No Comments' or something like that rather than drop them. we can discuss separate from this PR